.github/workflows/package-and-release.yaml

```diff
@@ -207,7 +207,12 @@ jobs:
           context: .
           file: dockerfile/ubuntu-20.04-amd64.Dockerfile
   release-docs:
-    needs: create-and-upload-draft-release
+    needs: [
+      get-params,
+      create-tag,
+      create-and-upload-draft-release
+    ]
+    if: ${{ needs.get-params.outputs.is-release-candidate =='true' && needs.get-params.outputs.should-create-release == 'true' }}
```

Comment:
Hmmm... we only upload docs on release candidates?

Reply:
The opposite - it needs to be not a full release! Thanks for catching!